### ISSUE TITLE
...ioc file for CubeMX

### ISSUE THREAD

**Issue by x893** [MikePhillyFlyers/T962A_Controller_TFT#1]
Hello,
awesome project !
I want port it to 32F746GDISCOVERY
https://www.st.com/en/evaluation-tools/32f746gdiscovery.html
Can you add Project.ioc (for CubeMX) ?

Best wishes

**Comment by MikePhillyFlyers** [comment]
Hi,

Sorry, wish I had this for you, but I only created an initial CubeMx (ioc) file when I first designed the board, but it was mainly to get some of the initial setup code generated.  

But I also used code snippets from other sample projects in the STM32H7 FW library from ST.  So as I moved forward, I only used CubeMx for reference, if I needed to generate a starting block of code for a peripheral I was adding, I would just run it, setup the periph, and have it generate the code.. then just cut n paste the code portion I needed..... But I never kept a CubeMx project updated with everything...

So all of the integration of FreeRTOS, STemWin, etc, I had to do by hand, as CubeMx didn't even have all of that for the H7 anyways. 

If you want an 32F7 Cubemx project, you'll prob. be better off just doing it from scratch...as I said above, I did so much manually and modified so much, anything I did create way back is not even close to what exists now...
